zulipterminal/ui_tools/boxes.py

```diff
@@ -82,6 +82,7 @@ def __init__(self, view: Any) -> None:
         self.send_next_typing_update = datetime.now()
         self.last_key_update = datetime.now()
         self.idle_status_tracking = False
+        self.sent_start_typing_status = False
```

Comment:
Could we improve this naming? Maybe "start_typing_status_sent"?

Reply:
I think the current name is fine.
It all depends on how you break that name while reading. e.g. currently with `sent_start_typing_status` you would break it something like: `sent` - `start_typing_status`, whereas with Sarthak's version of name, something like: `start_typing_status` - `sent`.

Plus, the current name is consistent with a similar naming above: `send_next_typing_update`. :)

Reply:
@zee-bit thanks for clarifying, that is what I had in mind.
I've also named it in a way that's easy to read, `sent_typing_start_status` meaning that we've "sent the `start` status". If it still seems like it can be improved, I'm happy to make it more readable. :)

Reply:
Let's amend the test associated with it as well in the corresponding `test_init`.